Gulpfile.babel.js

```diff
@@ -134,11 +134,12 @@ gulp.task("dist:js", () => {
 
 gulp.task("dist:noscript", () => {
   let sassPath = path.join(staticPrefix, "sass");
+  let nodeModules = "node_modules";
 
   return gulp.src(path.join(sassPath, "noscript.scss"))
     .pipe(sourcemaps.init())
     .pipe(
-      sass({ includePaths: [sassPath] })
+      sass({ includePaths: [sassPath, nodeModules] })
```

Comment:
Does this have the effect of including all node modules?

Reply:
Afaict, it has the effect of making all node modules searchable by an `@import` clause. if I `@import foo`, and there's no foo in `warehouse/static/sass/`, it will look for a foo in `node_modules/`.

If you'd rather we add specifically the right folder, we could do it. I believe we could also `@import ../../../node_module/normalize.css` but I don't like it at all.

Reply:
I think that's fine if it's only including explicitly imported modules in the bundle.